src/main/java/github/SsItAcademy/PrimeDigits/Main.java

```
@@ -0,0 +1,59 @@
+package github.SsItAcademy.PrimeDigits;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+public class Main {
+    public static void main(String[] args) {
+        long m = System.currentTimeMillis();
+        int a;
+        int b;
+        int c;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("введите диапозон через пробел");
+        a = sc.nextInt();
+        b = sc.nextInt();
+        System.out.println("введите количество потоков");
+        c = sc.nextInt();
+        ExecutorService executor = Executors.newFixedThreadPool(c);
+        int[] mass = new int[c + 1];
+        int step = (b - (a - 1)) / c;
+        mass[0] = a;
+        mass[1] = step;
+        for (int i = 2; i <= c; i++) {
+            mass[i] = mass[i - 1] + step;
+        }
+        mass[c]++;
+
+        ArrayList<ArrayList<Integer>> s = new ArrayList<>();
+        for (int i = 0; i < c; i++) {
+            s.add(new ArrayList<>());
+        }
+
+
+        for (int i = 0; i < c; i++) {
+            int one = 1;
+            if (i == c - 1)
+                one = 0;
+            executor.submit(new myThread(mass[i], mass[i + 1] - one, s.get(i)));
+        }
+        executor.shutdown();
+
+        try {
+            executor.awaitTermination(Long.MAX_VALUE, TimeUnit.NANOSECONDS);
+        } catch (InterruptedException e) {
```

Don`t use empty catches.
Please fix it.